po/fr.po

```diff
@@ -1286,9 +1278,9 @@ msgid "Go to a random unread article"
 msgstr "Lire un article au hasard"
 
 #: src/keymap.cpp:125
-#, fuzzy
 msgid "Open URL of article, or entry in URL view. Mark read."
-msgstr "Ouvrir l'article dans le navigateur et le marquer comme lu"
+msgstr ""
+"Ouvrir l'article ou l'élément dans le navigateur et le marquer  comme lu"
```

Comment:
A bit too many spaces here :)
```suggestion
"Ouvrir l'article ou l'élément dans le navigateur et le marquer comme lu"
```

Wouldn't be a problem for a non-TUI app, but Newsboat runs in a terminal with a (presumably) monospace font, so this little detail will be quite noticeable.